From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: MC_XiaoHei <xor7xiaohei@gmail.com>
Date: Sat, 5 Jul 2025 11:31:44 +0800
Subject: [PATCH] temp fix quick craft

caused by PaperMC/Paper#11765
should remove when PaperMC/Paper#11831 merged

diff --git a/net/minecraft/world/entity/LivingEntity.java b/net/minecraft/world/entity/LivingEntity.java
index e3673ababa41a4d65b87adbc6d79d53a196564b7..6c9a7ae3e9a340de35ca4dfcd486cfe5818d2aa5 100644
--- a/net/minecraft/world/entity/LivingEntity.java
+++ b/net/minecraft/world/entity/LivingEntity.java
@@ -3945,11 +3945,11 @@ public abstract class LivingEntity extends Entity implements Attackable, Waypoin
             return null;
         } else {
             double d = this.getEyeY() - 0.3F;
-            // Paper start
-            final ItemStack tmp = stack.copy();
-            stack.setCount(0);
-            stack = tmp;
-            // Paper end
+            // Paper start // Leaves start - wtf this for? it breaks quick craft
+            // final ItemStack tmp = stack.copy();
+            // stack.setCount(0);
+            // stack = tmp;
+            // Paper end // Leaves end
             ItemEntity itemEntity = new ItemEntity(this.level(), this.getX(), d, this.getZ(), stack);
             itemEntity.setPickUpDelay(40);
             if (includeThrower) {
